.drone.yml

```diff
@@ -0,0 +1,383 @@
+workspace:
+  base: /var/www/owncloud
+  path: apps/contacts
+
+branches: [master, release*, release/*]
+
+pipeline:
+  install-server:
```

Comment:
now we call this ``install-core``
look in passowrd_policy or files_texteditor for an example of all the "standard" stuff.